strapi/strapi
# [Blocks editor] Show i18n icon and enforce disabled state

### What does it do?

- Prevents editing a disabled field using the blocks editor
- Shows the i18n 🌍 icon next to a field label when it's localized

### Why is it needed?

To provide the behavior that users expect and match the functionality of the other fields.

### How to test it?

Create a new blocks field in a localized content type, make sure that you see the icon, and that you don't see it when the field is not localized.

Make a blocks field non-editable in the configure the view page. The field should now be disabled.
